twoopstracker/twoops/views.py

```diff
@@ -257,7 +259,7 @@ def get_queryset(self):
                 search_type = get_search_type(query)
                 if search_type == "raw":
                     query = refromat_search_string(query)
-                vector = SearchVector("content", "actual_tweet")
+                vector = SearchVector("content")
```

Comment:
Which contains most content? `content` or `actual_tweet`?

Reply:
@kilemensi 
`actual_tweet` does but it's confusing in the UI when you search for something and the content being displayed in the UI doesn't contain anything you searched for.

Reply:
Cool, can we then search content and owner.user_name and owner.name since they're visible?